File: app/src/main/java/dev/trindadedev/ui/components/dialog/LoadingDialog.java
package dev.trindadedev.ui.components.dialog;

import android.app.Dialog;
import android.content.Context;
import android.graphics.Color;
import android.graphics.drawable.ColorDrawable;
import android.os.Bundle;

import com.gero.newpass.databinding.DialogLoadingBinding;

public class LoadingDialog extends Dialog {

    private DialogLoadingBinding binding;
    private Context c;

    public LoadingDialog(Context context) {
        super(context);
        c = context;
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DialogLoadingBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        if (getWindow() != null) {
            getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
        }
    }
}

File: app/src/main/java/dev/trindadedev/ui/components/dialog/LoadingDialog.kt
package dev.trindadedev.ui.components.dialog

import android.app.Dialog
import android.content.Context
import android.os.Bundle

import com.geero.newpass.databinding.DialogLoadingBinding

class LoadingScreen(context: Context) : Dialog(context) {

    private lateinit var binding: DialogLoadingBinding
    private val c: Context = context

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DialogLoadingBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}
